RPI/scripts/run_rpi.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# *******************************************************************************
+# * Copyright 2013, by the California Institute of Technology.
+# * ALL RIGHTS RESERVED. United States Government Sponsorship
+# * acknowledged.
+# *
+
+DIRNAME="`dirname $0`"
+
+# Set BUILD_ROOT if unset or "" set the BUILD_ROOT to be the above dir
+if [ -z ${BUILD_ROOT} ]
+then
+    export BUILD_ROOT="`cd ${DIRNAME}/../..; pwd`"
+fi
+
+DEPLOY=`cd ${DIRNAME}/..; pwd;`
+${BUILD_ROOT}/Gds/bin/run_deployment.sh --deploy "${DEPLOY}" "$@"
```

Comment:
Does this line need to add the `-n` flag, as the Ground Station is typically not run on the RPI, and thus should not attempt to run the FSW.

Reply:
I think I'm confused, doesn't -n not run the app and only run the ground station/tcpserver?

Reply:
It does, I assumed you were running on the ground, not the pi.